paddle/fluid/imperative/partial_grad_engine.cc

```diff
@@ -152,8 +161,10 @@ static void GetGraphInfoBetweenTargets(
   target_vars = *input_target_grads;
 
   std::queue<std::pair<OpBase * /*op*/, OpBase * /*pending op*/>> op_queue;
+  std::unordered_set<std::pair<OpBase *, OpBase *>, HashPair> op_base_visited;
   for (auto &endpoint_op : endpoint_ops) {
     op_queue.emplace(endpoint_op, nullptr);
+    op_base_visited.insert(std::make_pair(endpoint_op, nullptr));
```

Comment:
Prefer to use `op_base_visited.emplace(endpoint_op, nullptr);` for the conciseness and the performance. This is just a suggestion.

Reply:
DONE